ReactNativeClient/lib/reducer.js

```diff
@@ -57,6 +57,8 @@ const defaultState = {
 	editorNoteStatuses: {},
 };
 
+const MAX_HISTORY = 200;
```

Comment:
It keeps the MAX_HISTORY length slice from the back.

Reply:
Ok so it really only applies to the back history, because the length of the forward history is naturally limited to the maximum length of the back history.  That's good.

Reply:
By "from the back", I mean that from the back of the array (preserving only the latest history)
I'm doing the slice for both forwardHistoryNotes and backwardHistoryNotes.

Reply:
That is fine.  What I meant is you probably don't have to apply this to the forward history, because it will never grow longer than the length of the backward history.  However it doesn't hurt, so no need to change.

In any case, a test case that checks it works would still be beneficial.